server/src/main/java/org/opensearch/index/remote/RemoteSegmentTransferTracker.java

```diff
@@ -525,6 +531,10 @@ void updateUploadTimeMsMovingAverageWindowSize(int updatedSize) {
         }
     }
 
+    public long getTotalUploadTimeInMs() {
+        return totalUploadTimeInMs;
+    }
```

Comment:
not being used, we can remove this.

Reply:
Added more UTs which are using these methods